app/api/dashboard/__tests__/summary-unscheduled.test.ts

```
@@ -0,0 +1,337 @@
+          select: jest.fn().mockReturnThis(),
+          in: jest.fn().mockReturnThis(),
+          eq: jest.fn().mockReturnThis(),
+          then: jest.fn().mockResolvedValue({ data: dailyAttendance, error: null }),
+        };
+      }
+      if (table === 'm_schools') {
+        return {
+          select: jest.fn().mockReturnThis(),
+          in: jest.fn().mockReturnThis(),
+          then: jest.fn().mockResolvedValue({ data: [], error: null }),
+        };
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)